src/stats/performance.ts

```diff
@@ -27,10 +27,22 @@ const calcApplePodcastPerformanceQuarters = function (
 
     const maxListeners = Math.max(...performanceValues)
 
-    if (performanceValues.length < 4) {
-        throw Error('too little data to calculate quarters')
+    // apple LTR data can contain q weird long tail which is a way longer than the episode itself
+    // filter out longtail if it is lower than the defined threshold (percent)
+    // start from the end of the array
+    const longtailThreshold = 0.05
+    const longtailThresholdValue = maxListeners * longtailThreshold
+    let longtailIndex = performanceValues.length - 1
+    while (
+        longtailIndex > 0 &&
+        performanceValues[longtailIndex] <= longtailThresholdValue
+    ) {
+        longtailIndex--
     }
+    // remove longtail from the data array
+    performanceValues.splice(longtailIndex + 1)
```

Comment:
We could put that part into a separate function to make testing easier. Let's maybe add a few tests for the trimming algorithm, like `[], [1], [-1], [0.1, 1000], [1000, 0.01, 4000], [1000, 0.1, 0.1, 0.1]`

Reply:
something like `cutAtThreshold(series, threshold)`